### PR TITLE
Add sample Kafka collectd config

### PR DESCRIPTION
**Pull request by RajivSignal**

### CODE REVIEW

File: managed_config/20-kafka-conf [hunk 1]
```diff
@@ -0,0 +1,262 @@
+# Install:
+#   Please follow the install instructions of 10-jmx.conf
+
+# Documentation:
+#   http://wiki.apache.org/cassandra/Metrics
```
Comment (cep21) [inline managed_config/20-kafka-conf]: FIx link

File: managed_config/20-kafka-conf [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+                Type "gauge"
+                Table false
+                Attribute "99thPercentile"
+                InstancePrefix "kafka.offset-fetch.total-time.99th"
+            </Value>
+        </MBean>
+
+        <Connection>
+            ServiceURL "service:jmx:rmi:///jndi/rmi://localhost:7099/jmxrmi"
+            Host "i-9ed4af72_kafka"
```
Comment (cep21) [inline managed_config/20-kafka-conf]: ??

File: managed_config/20-kafka-conf (hunk 1 above)
Comment (RajivSignal) [inline managed_config/20-kafka-conf]: Fixed this.

Comment (cep21) [inline managed_config/20-kafka-conf]: Where?

File: managed_config/20-kafka-conf (hunk 2 above)
Comment (RajivSignal) [inline managed_config/20-kafka-conf]: Changed to Host "testkafkaserver"

File: managed_config/20-kafka-conf [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+        <Connection>
+            ServiceURL "service:jmx:rmi:///jndi/rmi://localhost:7099/jmxrmi"
+            Host "testkafkaserver"
+            Collect "classes"
+            Collect "garbage_collector"
+            Collect "memory-heap"
+            Collect "memory-nonheap"
+            Collect "memory_pool"
+            Collect "threading"
+            Collect "operating_system"
```
Comment (cep21) [inline managed_config/20-kafka-conf]: Where is ```operating_system``` defined?

Comment (wt) [inline managed_config/20-kafka-conf]: It's defined in the sf-salt-files.git/collectd/conf/java-jmx-generic.conf in our salt configs. It doesn't seem to exist in this repo.

Comment (RajivSignal) [inline managed_config/20-kafka-conf]: I'll remove "operating_system" and "threading" for now.

**Comment by mpetazzoni** [comment]
For the record it would have been good to squash before merging here.

**Pull request merged**